app/views/planning_applications/assessment/tasks/index.html.erb

```diff
@@ -32,6 +32,12 @@
 
       <%= render "assess_against_legislation" %>
 
+      <% unless Bops.env.production? %>
+        <% unless @planning_application.application_type.name == "planning_permission" %>
```

Comment:
How come planning permission specifically is excluded? Is this something we can check application type features for instead?

Reply:
Just a copy + paste of the previous logic. We don't currently capture this in application type features so we'd need to add something to the choose features page. I think this can be done in a following PR